spec/requests/users_spec.rb

```diff
@@ -71,3 +71,21 @@
     end
   end
 end
+
+describe 'User Requests', type: :request do
+  let(:endpoint) { "/api/users/#{user.id}" }
+  let(:user) { create(:user) }
+  let(:update_params) do
+    {
+      part: 'first'
+    }
+  end
+  describe 'PUT /api/users/:id' do
+    it 'successfully updates a user\'s part' do
+      put endpoint, params: update_params.to_json, headers: authenticated_header(user)
+
+      expect(response).to have_http_status(200)
+      user.part.should eq(User.parts[update_params['part']])
```

Comment:
Thanks for the heads up. That's been fixed locally. I just haven't pushed in a while. RSpec has nice deprecations when you run the tests. Thanks for the resources!